his/core/configuring-the-as-400-for-5250-access2.md

```diff
@@ -47,19 +47,20 @@ To configure these parameters, you must be logged on with administrative privile
+
+You can connect to an IBM i server for 5250 emulation even if the IBM i server doesn't have PC support. For this task, you must create the mode QPCSUPP on the IBM i, along with other resources that would have been created by PC support. A recommended way to do this is to create the QPCSUPP mode, create the QWCPCSUP class, and add the class QWCPCSUP to the system QCMN. For details about how to create these resources, see the online i Series documentation for the related commands: **crtmodd**, **crtcls**, and **addrtge**.
 
 ## Change session limits with a single mode name  
 
-If the i Series requires different session limits on each LU, while the mode name stays the same, you must change the device description attached to the controller to reflect the correct number of sessions allowed for the LUs. For example, with a 5250 emulator, the mode name must be QPCSUPP. If you change the Parallel Session Limit on QPCSUPP for one LU-LU pair, all other LU-LU pairs that use the QPCSUPP mode will also be affected. Therefore, to change the session limit on some LUs but not others, you must change it at the i Series.
+If the IBM i server requires different session limits on each LU, while the mode name stays the same, you must change the device description attached to the controller to reflect the correct number of sessions allowed for the LUs. For example, with a 5250 emulator, the mode name must be QPCSUPP. If you change the Parallel Session Limit on QPCSUPP for one LU-LU pair, all other LU-LU pairs that use the QPCSUPP mode will also be affected. Therefore, to change the session limit on some LUs but not others, you must change it at the IBM i server.
 
-The simplest way to get the correct number of sessions for the LUs on the i Series is to autocreate the controllers and devices, and then change the device description attached to the controller, so that the session limit is decreased. This is done by using the Change Device Description (**chgdevappc**) command. It needs to be done for each device description that needs to have different session limits. The i Series device settings that should be modified are Maximum Sessions (**maxssn**) and Maximum Conversations (**maxcnv**).  
+The simplest way to get the correct number of sessions for the LUs on the IBM i is to autocreate the controllers and devices, and then change the device description attached to the controller, so that the session limit is decreased. This is done by using the Change Device Description (**chgdevappc**) command. It needs to be done for each device description that needs to have different session limits. The IBM i device settings that should be modified are Maximum Sessions (**maxssn**) and Maximum Conversations (**maxcnv**).  
```

Comment:
```suggestion
For the simplest way to get the correct number of sessions for the LUs on the IBM i. automatically create the controllers and devices. Then, change the device description attached to the controller, so that the session limit is decreased. For this task, use the Change Device Description command (**chgdevappc**). You must complete this task for each device description that needs to have different session limits. Modify the IBM i device settings for Maximum Sessions (**maxssn**) and Maximum Conversations (**maxcnv**).  
```